static/panes/compiler.ts

```diff
@@ -3477,18 +3492,23 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         }
 
         try {
-            const asmHelp = await this.getAsmInfo(word.word, this.compiler?.instructionSet ?? '');
-            if (asmHelp) {
-                this.alertSystem.alert(opcode + ' help', asmHelp.html + appendInfo(asmHelp.url), () => {
-                    ed.focus();
-                    ed.setPosition(pos);
-                });
-            } else {
-                this.alertSystem.notify('This token was not found in the documentation. Sorry!', {
-                    group: 'notokenindocs',
-                    alertClass: 'notification-error',
-                    dismissTime: 5000,
-                });
+            if (this.compiler?.supportsAsmDocs) {
```

Comment:
I think it makes sense to wrap everything here in a `supportsAsmDocs` condition

Reply:
Works for me, yes